scripts/validate_phone_numbers.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import json
+import sys
+
+# usage: AWS_PROFILE=notify-production python validate_phone_numbers.py numbers.txt
+# numbers.txt should have one phone number per line.
+
+# example Hasura query to get numbers.txt
+#
+#     select distinct "to"
```

Comment:
It would be better to get from the `notifications.normalised_to` column to get a standard format.